corehq/apps/data_cleaning/views/columns.py

```diff
@@ -33,7 +33,7 @@ def get_context_data(self, column_form=None, **kwargs):
 
     def _trigger_clean_form_refresh(self, response):
         response['HX-Trigger'] = json.dumps({
-            'dcCleanFormRefresh': {
+            'dcEditFormRefresh': {
```

Comment:
I assume the `dc` here is short for "data cleaning". If we're updating "data cleaning" to other names, does it also make sense to drop the `dc` here?

Reply:
i'm kind of ambivalent about this. I always like to namespace events and `data_cleaning` is still the name of the module